images/image.mk

```diff
@@ -127,12 +127,7 @@ PRUNE_KEEP ?= 24
 PRUNE_DRYRUN ?= 0
 
 CACHE_DATE_FORMAT := "%Y-%m-%d.%H%M%S"
-ifeq ($(UNAME_S),Linux)
-CACHE_PRUNE_DATE := $(shell date -u --date="$(PRUNE_HOURS) hours ago" +"$(CACHE_DATE_FORMAT)")
-endif
-ifeq ($(UNAME_S),Darwin)
-CACHE_PRUNE_DATE := $(shell date -u -v -$(PRUNE_HOURS)H +"$(CACHE_DATE_FORMAT)")
-endif
+CACHE_PRUNE_DATE := $(shell export TZ="UTC+$(PRUNE_HOURS)"; date +"$(CACHE_DATE_FORMAT)")
```

Comment:
why is it `UTC+$(PRUNE_HOURS)` and not `UTC-$(PRUNE_HOURS)`?

Reply:
not sure TBH, but it works
```
bash-4.4$ TZ="UTC" date
Sat Nov 18 00:15:12 UTC 2017
bash-4.4$ TZ="UTC+24" date
Fri Nov 17 00:15:17 UTC 2017
```

Reply:
odd. from https://www.w3.org/TR/NOTE-datetime:

```Times are expressed in local time, together with a time zone offset in hours and minutes. A time zone offset of "+hh:mm" indicates that the date/time uses a local time zone which is "hh" hours and "mm" minutes ahead of UTC. A time zone offset of "-hh:mm" indicates that the date/time uses a local time zone which is "hh" hours and "mm" minutes behind UTC```

Reply:
oh its the reverse. if the timezone its ahead of UTC then the current time is behind. confusing.